### PR TITLE
179D - Set People Fraction Sensible

### PR DESCRIPTION
**Pull request by jmarrec**
Override space_type_apply_internal_loads to apply People Fraction Sensible per ACM rules instead of letting E+ AutoCalculate it

### CODE REVIEW

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb [hunk 1]
```diff
@@ -43,4 +43,19 @@ def space_type_get_standards_data(space_type, extend_with_2007: true, throw_if_n
 
     return space_type_properties
   end
+
+  # NOTE: 179D overrides it to set the people fraction sensible per ACM rules instead of letting E+ AutoCalculate it
+  def space_type_apply_internal_loads(space_type, set_people, set_lights, set_electric_equipment, set_gas_equipment, set_ventilation, set_infiltration)
+    super(space_type, set_people, set_lights, set_electric_equipment, set_gas_equipment, set_ventilation, set_infiltration)
```
Comment (jmarrec) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb:49]: Call super method

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb [hunk 2]
```diff
@@ -43,4 +43,19 @@ def space_type_get_standards_data(space_type, extend_with_2007: true, throw_if_n
...
+
+    if set_people
+      data = space_type_get_standards_data(space_type, extend_with_2007: false, throw_if_not_found: true)
+      people_frac_sensible = data['occupancy_fraction_sensible']
+      space_type.people.sort.each do |inst|
+        definition = inst.peopleDefinition
+        definition.setSensibleHeatFraction(people_frac_sensible)
+      end
+    end
+  end
```
Comment (jmarrec) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb:59]: Then patch it

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/data/179d_ashrae_90_1_2007.spc_typ.json [hunk 3]
```diff
@@ -45,6 +45,7 @@
       "occupancy_schedule": "Nonres_Occ_Sch",
       "occupancy_activity_schedule": "ACM_FullServiceRestaurant_ACTIVITY_SCH",
       "is_residential": null,
+      "occupancy_fraction_sensible": 0.451,
```
Comment (jmarrec) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/data/179d_ashrae_90_1_2007.spc_typ.json:48]: new field added on the google spreadsheet.

ACM gives Sensible Heat Gain and Latent Heat Gain (Btu/h).
Sensible + Latent = Total, convert that to Watts => this is the value of the "occupancy_activity_schedule"

Sensible / Total = occupancy_fraction_sensible

**Review by brianlball — APPROVED** [review]
i like

**Pull request merged**